src/test/java/com/marizueva/laboratory/hw4/utils/DriverManager.java

```diff
@@ -0,0 +1,26 @@
+package com.marizueva.laboratory.hw4.utils;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.util.concurrent.TimeUnit;
+
+public class DriverManager {
```

Comment:
Class should be `final` and has `private` constructor

Reply:
fixed